### PR TITLE
Feat/basics

### PR DESCRIPTION
**Pull request by alinepickler**

### CODE REVIEW

File: src/components/footer/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    url: "https://reactjs.org/docs/typechecking-with-proptypes.html"
+  },
+  {
+    text: "Create react app",
+    url: "https://github.com/facebook/create-react-app/"
+  },
+  { text: "github", url: "https://github.com/alinepickler/my-project.git" }
+];
+
+const first = additionalMenu[0];
```
Comment (iamstarkov) [inline src/components/footer/index.js]: this is a dead code, we don't need it anymore. pls remove

File: src/components/header/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import { Menu } from "../";
+import "./styles.css";
+
+const mainMenu = [
+  { text: "New", url: "https://news.ycombinator.com/newest" },
+  { text: "Show", url: "https://news.ycombinator.com/show" },
+  { text: "Submit", url: "https://news.ycombinator.com/submit" }
+];
+
+const first = mainMenu[0];
```
Comment (iamstarkov) [inline src/components/header/index.js]: same here

**Review by iamstarkov — CHANGES_REQUESTED** [review]
minor: some dead code needs to be removed

File: src/components/footer/index.js (hunk 1 above)
Comment (alinepickler) [reply src/components/footer/index.js]: Fixed =)

File: src/components/header/index.js (hunk 2 above)
Comment (alinepickler) [reply src/components/header/index.js]: Fixed =)

**Review by iamstarkov — APPROVED** [review]
👍 looks good to me!

**Pull request merged**